### ISSUE TITLE
can not get plugin 'UndoRedo'

### ISSUE THREAD

**Issue by HaroldStruwig** [handsontable/handsontable#4192]
### Demo
http://jsfiddle.net/kkdhjqtv/1/

### Your environment
* Handsontable version: 1.10.2 PRO

**Comment by AMBudnik** [comment]
You're right - we call it a plugin but you cannot access it via `getPlugin` method (more like this http://jsfiddle.net/handsoncode/ucux0e80/)

**Comment by AMBudnik** [comment]
I have changed the title to describe our goal. The issue is waiting for suggestions and dev review.

**Comment by AMBudnik** [comment]
I was just informed that we first need to rewrite the plugin to ES6. This action hasn't been scheduled yet. However, I highly encourage to check our Roadmap for updates https://trello.com/b/PztR4hpj/handsontable-roadmap

**Comment by wojciechczerniak** [comment]
@AMBudnik I think that undo/redo is now accessible through getPlugin, since v8?

**Comment by AMBudnik** [comment]
Hi @wojciechczerniak 

please check this example where I'm loading v 8.1.0 https://jsfiddle.net/vy8Lmw5t/1/ 

I change the dataset

`hot.setDataAtCell(0, 0, 'test');`

and then try to undo this operation

```
//hot.getPlugin('undoRedo').undo(); //doesn't work
hot.undoRedo.undo(); //works
```

**Comment by AMBudnik** [comment]
Hi @HaroldStruwig 

This issue is no longer replicable; we can call the UndoRedo plugin via the `getPlugin()` method. Here is a demo you can test: https://jsfiddle.net/w1hpd4c5/1/ 

Sorry for the delay. It was changed so long ago, but we lost this ticket in the reference.
